index.js

```diff
@@ -0,0 +1,32 @@
+const taskList = () => {
+  console.log(`
+  ===================
+  現在持っているのタスク
+  ===================
+  `);
+};
+taskList();
+
+let currentTask = [
+  ['机を片付ける', '掃除'],
+  ['牛乳を買う', '買い物'],
+  ['散歩する', '運動']
+];
```

Comment:
Object の配列にした方が扱いやすいと思います。

```suggestion
const currentTask = [
  { content: '机を片付ける', genre : '掃除' },
  { content: '牛乳を買う', genre: '買い物' },
  { content: '散歩する', genre: '運動' }
]
```